src/routes/api/albums.js

```diff
@@ -19,4 +39,20 @@ router.post('api.albums.create', '/', async (ctx) => {
   }
 });
 
+router.patch('api.albums.update', '/:id', async (ctx) => {
+  const { cloudinary } = ctx.state;
+  try {
+    const album = await ctx.orm.album.findByPk(ctx.params.id);
```

Comment:
Ya deberías tener disponible `ctx.state.album` en este punto, por lo que esto sólo podría ser:
```suggestion
    const { album } = ctx.state;
```